source/scripts/app.js

```diff
@@ -601,8 +601,71 @@ function bindCookbookCardButtons(card) {
   });
 
   openButton.addEventListener("click", () => {
-    // TODO set up cookbook page
     router.navigate("single-cookbook");
+    populateSingleCookbook(card.cookbook);
```

Comment:
You'll want to populate the recipes before you navigate to the single cookbook, because for cookbooks with more recipes, it'll take more time to load in the recipes, and if the user is on the single cookbook page while the recipes are populating, it'll cause screen flicker. I'm thinking of adding loading states for these type of events in Sprint 2, so that the user isn't wondering why the buttons aren't redirecting them immediately.
```suggestion
    await populateSingleCookbook(card.cookbook);
    router.navigate("single-cookbook");
```